doc/api/vm.md

```diff
@@ -135,8 +174,7 @@ changes:
     description: The `breakOnSigint` option is supported now.
 -->
 
-* `contextifiedObject` {Object} A [contextified][] object as returned by the
-  `vm.createContext()` method.
+* `context` {vm.Context} A [`vm.Context`][] instance.
```

Comment:
Same here. At a minimum, a union type should be used saying that the function accepts both a `vm.Context` and a contextified object.